README.md

```diff
@@ -200,3 +200,59 @@ You can run Celery workers for Concent with:
 ``` bash
 concent_api/celery worker --app concent_api --loglevel info --queues concent,conductor,verifier
 ```
+
+### Deploying GNTDeposit contract
+
+To be able to run e2e tests locally, that contains transactions on blockchain,
+Concent needs separate GNTDeposit contract to work without errors.
+
+1. Get [GNTDeposit.sol](https://github.com/golemfactory/golem-contracts/blob/master/contracts/GNTDeposit.sol)
+from `golem-contracts` repository.
+
+2. Fallow this [tutorial](https://medium.com/swlh/deploy-smart-contracts-on-ropsten-testnet-through-ethereum-remix-233cd1494b4b)
+to deploy GNTDeposit.sol on `rinkeby` ethereum testnet.
+
+    It involves installing `MetaMask` on your browser, creating ethereum account with some testnet ETH and copy
+    GNTDeposit.sol with all dependencies to single file in solidity IDE - `remix`. You can use
+    [solidity-flattener](https://github.com/BlockCatIO/solidity-flattener) to do so.
+
+3. There are four variables needed to deploy new GNTDeposit contract:
+
+    - `_token` - GNTB contract address. For all testnet environments, it should be hardcoded to
+    `0x123438d379BAbD07134d1d4d7dFa0BCbd56ca3F3`.
+
+    - `_concent` - Concent ethereum address. It can be the same for all testnet environments
+    `0xa5f3cd35385755e64b47f20a5d7334d7013f8015`.
```

Comment:
Please explain what this value means. Is it the address of the testnet Concent contract deployed by Golem?

Reply:
Nope, it's Concent ethereum address(like wallet), not contract. Also the same for all our clusters. I don't remember 100% what is it use for exactly. It's not use to store eth or gnt, that's for sure. I think it's just a reference for the rest of hardcoded contract dresses from https://github.com/golemfactory/golem/blob/develop/golem/config/environments/testnet.py#L41

Reply:
In that case it must be the address where the contract stores GNT corresponding to its deposits?

Please add what you know in the description. And what you don't know - please ask Igor or someone else from Golem. We're working with that contract so we're supposed to know that. We cannot just guess.